docs/zkapps/how-to-deploy-a-zkapp.mdx

```diff
@@ -46,28 +48,34 @@ You see the following output:
 $ zk config
 
 Add a new network:
-✔ Choose a name (can be anything): · berkeley
-✔ Set URL to deploy to: · https://proxy.berkeley.minaexplorer.com/graphql
+✔ Create a name (can be anything): · berkeley
```

Comment:
Should we change this Berkeley deployment info to `devnet` now? :D

Reply:
I think it should be covered by generalized work on https://github.com/o1-labs/docs2/issues/912.
[Internal discussion](https://o1-labs.slack.com/archives/C028Q27R8UC/p1714226290308199?thread_ts=1713980495.202589&cid=C028Q27R8UC).

Reply:
I think we should deploy everything in the docs to `devnet`

Reply:
As I mentioned in reply above:
- I was thinking that changing stuff like this (names, links, etc.) would be good to do in one separate PR to find and replace all entries not as part of individual files.
But can do it for these specific files if you think it should be done like this.

Reply:
Will be done as part of the work on https://github.com/o1-labs/docs2/issues/916.

Reply:
That approach is fine with me. I would just make sure all the necessary code that  is changed in #916 works correctly with devnet.